src/backend/utils/adt/formatting.c

```diff
@@ -2973,7 +2973,6 @@ DCH_to_char(FormatNode *node, bool is_interval, TmToChar *in, char *out, Oid col
 				s += strlen(s);
 				break;
 			case DCH_RM:
-				/* FALLTHROUGH */
```

Comment:
I think this change doesn't fix warning. If I'm right, explain please the purpose of this change in the description or revert it

Reply:
See my answer to the first comment